io.github.jordanl2.ModularCalculator.yml

```diff
@@ -0,0 +1,30 @@
+app-id: io.github.jordanl2.ModularCalculator
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+default-branch: stable
+command: modularcalculator
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
```

Comment:
why does it need full home access?

Reply:
It needs to be able to save and open files, so home access seemed like a sensible minimum level of file system access.

Reply:
maybe `xdg-documents` instead?

Reply:
Ok, committed